modules/core/04-channel/keeper/timeout_test.go

```diff
@@ -247,46 +248,120 @@ func (suite *KeeperTestSuite) TestTimeoutPacket() {
 }
 
 // TestTimeoutExectued verifies that packet commitments are deleted on chainA after the
-// channel capabilities are verified.
+// channel capabilities are verified. In addition, the test verified that the channel state
+// after a timeout is updated accordingly.
 func (suite *KeeperTestSuite) TestTimeoutExecuted() {
 	var (
 		path    *ibctesting.Path
 		packet  types.Packet
 		chanCap *capabilitytypes.Capability
 	)
 
-	testCases := []testCase{
-		{"success ORDERED", func() {
-			path.SetChannelOrdered()
-			suite.coordinator.Setup(path)
+	testCases := []struct {
```

Comment:
completely revamped this test to allow checking for how the state of affairs is post timeoutexecuted (also, the error returned). I think this is a way better approach than what previously existed though its unfortunate it added an additional diff to check.